pre-pages/pgql-2.0-spec.md

```diff
@@ -1260,9 +1361,16 @@ For example:
 {% include image.html file="example_graphs/financial_transactions.png" %}
 
 ```sql
-SELECT label(n), n.*
+--PGQL
+SELECT ID(n), n.*
```

Comment:
the output below still shows label(n) instead of ID(n).

But I think best may be to not have queries that return IDs since they look so different in PGQL vs SQL property graphs. I think I managed to avoid such queries till now. 

So I would say maybe just remove that column and keep the other two columns.